GrammarNazi.Core/GrammarNazi.Core.csproj

```diff
@@ -19,7 +19,7 @@
     <PackageReference Include="System.ServiceModel.Http" Version="4.9.0" />
     <PackageReference Include="System.ServiceModel.NetTcp" Version="4.9.0" />
     <PackageReference Include="System.ServiceModel.Security" Version="4.9.0" />
-    <PackageReference Include="Telegram.Bot" Version="17.0.0" />
+    <PackageReference Include="Telegram.Bot" Version="18.0.0" />
 	<PackageReference Include="Telegram.Bot.Extensions.Polling" Version="1.0.2" />
```

Comment:
We need to remove this `Telegram.Bot.Extensions.Polling` package

Reply:
And then, we need to update this file TelegramUpdateHandler.cs that is using the old `IUpdateHandler `https://github.com/nminaya/grammar-nazi-bot/blob/master/GrammarNazi.Core/Utilities/TelegramUpdateHandler.cs